lib/galaxy/webapps/tool_shed/app.py

```diff
@@ -10,9 +10,11 @@
 from galaxy.openid.providers import OpenIDProviders
 from galaxy.util.dbkeys import GenomeBuilds
 from galaxy.web import security
+from galaxy.config import configure_logging
```

Comment:
@nsoranzo I should be good now.
![img_20160805_103241](https://cloud.githubusercontent.com/assets/1814954/17439905/146763e0-5af8-11e6-9c90-360250c8ecb8.jpg)